platforms/hyperledger-fabric/charts/fabric-peernode/templates/configmap.yaml

```diff
@@ -103,24 +101,4 @@ metadata:
     app.kubernetes.io/release: {{ .Release.Name }}
     app.kubernetes.io/managed-by: helm
 data:
-  core.yaml: {{ .Files.Get "files/core.yaml" | nindent 8 | quote }}
-{{ end }}
-{{- $orderercrt := .Files.Get "files/orderer.crt" }}
```

Comment:
orderercrt should be created, it is needed for peer cli and chaincodes

Reply:
Done